aodh/server.sls

```diff
@@ -1,4 +1,22 @@
 {%- from "aodh/map.jinja" import server with context %}
-{%- if server.enabled %}
+{%- if server.enabled and server.version == "mitaka" %}
```

Comment:
No, {%- if server.enabled %} is common pattern, but Aodh is supported only still Mitaka and there is not Aodh on earlier openstack versions.